ldap_fluff.gemspec

```
@@ -16,10 +16,10 @@ Gem::Specification.new do |s|
   s.email    = %w[jomara@redhat.com elobatocs@gmail.com pchalupa@redhat.com
                   komidore64@gmail.com mhulan@redhat.com dominic@cleal.org]
 
-  s.required_ruby_version = '>= 2.4.0'
+  s.required_ruby_version = '>= 2.7', '< 4'
 
-  s.add_dependency('activesupport')
-  s.add_dependency('net-ldap', '>= 0.11')
-  s.add_development_dependency('minitest')
-  s.add_development_dependency('rake')
+  s.add_dependency('activesupport', '~> 7.1', '>= 7.1.2')
+  s.add_dependency('net-ldap', '~> 0.19.0')
```

I wonder why it needs to be this strict. Technically semver says that every 0.x version can be breaking, but in practice I'd trust this and only add an upper bound:
```suggestion
  s.add_dependency('net-ldap', '>= 0.11', '< 1')
```